types/mongoose-delete/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "only-arrow-functions": false,
```

Comment:
New packages should not override any lint rules. They should only extend `dtslint/dt.json`.